qa/tasks/ceph.py

```diff
@@ -610,195 +600,258 @@ def cluster(ctx, config):
-            coverage_dir,
-            'ceph-authtool',
-            '--create-keyring',
-            keyring_path,
-        ],
-    )
-    ctx.cluster.only(firstmon).run(
-        args=[
-            'sudo',
-            'adjust-ulimits',
-            'ceph-coverage',
-            coverage_dir,
-            'ceph-authtool',
-            '--gen-key',
-            '--name=mon.',
-            keyring_path,
-        ],
-    )
-    ctx.cluster.only(firstmon).run(
-        args=[
-            'sudo',
-            'chmod',
-            '0644',
-            keyring_path,
-        ],
-    )
-    (mon0_remote,) = ctx.cluster.only(firstmon).remotes.keys()
-    monmap_path = '{tdir}/{cluster}.monmap'.format(tdir=testdir,
-                                                   cluster=cluster_name)
-    fsid = create_simple_monmap(
-        ctx,
-        remote=mon0_remote,
-        conf=conf,
-        mons=mons,
-        path=monmap_path,
-        mon_bind_addrvec=config.get('mon_bind_addrvec'),
-    )
-    ctx.ceph[cluster_name].fsid = fsid
-    if not 'global' in conf:
-        conf['global'] = {}
-    conf['global']['fsid'] = fsid
+        coverage_dir = '{tdir}/archive/coverage'.format(tdir=testdir)
 
-    default_conf_path = '/etc/ceph/{cluster}.conf'.format(cluster=cluster_name)
-    conf_path = config.get('conf_path', default_conf_path)
-    log.info('Writing %s for FSID %s...' % (conf_path, fsid))
-    write_conf(ctx, conf_path, cluster_name)
-
-    log.info('Creating admin key on %s...' % firstmon)
-    ctx.cluster.only(firstmon).run(
-        args=[
-            'sudo',
-            'adjust-ulimits',
-            'ceph-coverage',
-            coverage_dir,
-            'ceph-authtool',
-            '--gen-key',
-            '--name=client.admin',
-            '--cap', 'mon', 'allow *',
-            '--cap', 'osd', 'allow *',
-            '--cap', 'mds', 'allow *',
-            '--cap', 'mgr', 'allow *',
-            keyring_path,
-        ],
-    )
+        log.info('Setting up %s...' % firstmon)
+        ctx.cluster.only(firstmon).run(
+            args=[
+                'sudo',
+                'adjust-ulimits',
+                'ceph-coverage',
+                coverage_dir,
+                'ceph-authtool',
+                '--create-keyring',
+                keyring_path,
+            ],
+        )
+        ctx.cluster.only(firstmon).run(
+            args=[
+                'sudo',
+                'adjust-ulimits',
+                'ceph-coverage',
+                coverage_dir,
+                'ceph-authtool',
+                '--gen-key',
+                '--name=mon.',
+                keyring_path,
+            ],
+        )
+        ctx.cluster.only(firstmon).run(
+            args=[
+                'sudo',
+                'chmod',
+                '0644',
+                keyring_path,
+            ],
+        )
+        (mon0_remote,) = ctx.cluster.only(firstmon).remotes.keys()
+        fsid = create_simple_monmap(
```

Comment:
I assume the only change here to solely bumping up the indentation.

Reply:
> I assume the only change here to solely bumping up the indentation.

Yes, I will change that